### ISSUE TITLE
Observatory_usecase1

### ISSUE THREAD

**Issue by ChristinaB** [Freshwater-Initiative/Observatory#16]
I had to copy ogh_meta.json from ogh to Observatory-1 folder in order to run

```
meta_file = dict(ogh.ogh_meta())
sorted(meta_file.keys())
```

is this designed to have ogh_meta.json in the same folder as where the Notebook is running or do you always want it to live in ogh?

**Comment by ChristinaB** [comment]
Is t1 or this function used?  Please advise if needed or how to fix.
![image](https://user-images.githubusercontent.com/4108369/38145933-9e6dc92a-3400-11e8-8b09-30f3ad948650.png)

**Comment by ChristinaB** [comment]
It would see that this definately needs to run from a 'Home' HydroShare location, correct?
![image](https://user-images.githubusercontent.com/4108369/38146002-dab767b0-3400-11e8-8c5d-2cdd4e6c0687.png)

**Comment by ChristinaB** [comment]
I resolved the issues above by using conda install ogh --yes and executing the notebooks from this HydroShare resource: https://www.hydroshare.org/resource/4ac20993aa244c08baa9e5965dd6b810/

I tested the code with three different shapefiles and broke the big map:  
![image](https://user-images.githubusercontent.com/4108369/38152721-e59a53e6-341d-11e8-8ad2-693bb2339944.png)  

Also, does list of watershed names in Section 4  
`listofwatershednames = ['Skagit River','Sauk-Suiattle River','Upper Forks']`
need to be the same as 
those listed in the dissolving step?

`# dissolve polygons by watershed
sk['watershed'] = 'Skagit River'
sa['watershed'] = 'Sauk-Suiattle River'
uf['watershed'] = 'Upper Sauk Forks'`

Request: Let's add subnumbering for sections so that each cell block has a immutable reference we can use in the paper/appendix.

**Comment by ChristinaB** [comment]
I don't think you should spend time fixing the big map, but I may not include it in these example notebooks, just use in the paper.

**Comment by ChristinaB** [comment]
My OGH doesn't have griddedCellGradient or mappingfileSummary

**Comment by jphuong** [comment]
I think we resolved this issue. The notebooks were not calling the conda library ogh.

**Comment by jphuong** [comment]
I'm not sure how the subnumbering soluton would be applied onto the figures or maintained throughout the use-cases. Seems like it would be easier to organize that outside of the notebooks.

The big map contains hardcoding, and I agree that it should be made scalable.
 - [x] The hawks-eye map should have scalable pointer references, fontsize, and scale map sizing according to the basemap projections.

**Comment by jphuong** [comment]
No, the mappingfileSummary() parameter listOfWatershedNames is purely aethetic for the output table column labels. 

It uses the mappingfiles as inputs; it does not rely on the watershed shapefiles or its dissolved polygon shapefiles. The dissolved polygon shapefiles are mainly used for plotting landscape maps without subpolygons outlines.
